test/Cell.test.js

```diff
@@ -1,4 +1,8 @@
-const s2 = require('bindings')('s2');
+// magic incantation from step 3 @ https://github.com/mapbox/node-pre-gyp#readme
+const binary = require('node-pre-gyp');
+const path = require('path');
+var binding_path = binary.find(path.resolve('./package.json'));
+const s2 = require(binding_path);
```

Comment:
hmm gotta say I'm not a fan of this expansion for our tests - does the old way not work anymore? what are the path differences?

Reply:
I will try to make it simpler, stand by

Reply:
see above for reason. I can't really make this any simpler.